docs/zh-CN/sql-reference/sql-functions/date-time-functions/convert_tz.md

```diff
@@ -25,14 +25,12 @@ under the License.
 -->
 
 # convert_tz
-
-## Syntax
+## description
+### Syntax
```

Comment:
why？

Reply:
Thanks for reminding. It has been modified